content/applications/productivity/data_cleaning.rst

```diff
@@ -0,0 +1,209 @@
+matching conditions within the records set by the :ref:`deduplication rules
+<data_cleaning/deduplication-rules>`.
+
+.. image:: data_cleaning/data-cleaning-duplicates.png
+   :align: center
+   :alt: Deduplication dashboard in the Data Cleaning application.
+
+The :guilabel:`RULE` sidebar lists each of the active deduplication rules, with the total number of
+duplicates detected displayed beside each rule.
+
+By default, the :guilabel:`All` rule is selected. Records display grouped by their rule, with a
+:guilabel:`Similarity` rating (out of 100%) in the list view with the following columns:
+
+- :guilabel:`Created On`: the date and time the original record was created.
+- :guilabel:`Name`: the name or title of the original record.
+- :guilabel:`Field Values`: the original record's values for the fields used to detect duplicates.
+- :guilabel:`Used In`: lists other models referencing the original record.
+- :guilabel:`ID`: the original record's unique ID.
+- :guilabel:`Is Master`: the duplicates are merged into the *master* record. There can be **one**
+  master record in a grouping of similar records.
+
+Select a specific rule in the :guilabel:`RULE` sidebar to filter the duplicate records.
+
+.. _data-cleaning/merge-records:
+
+Merge duplicate records
+-----------------------
+
+To merge records, first choose a *master* record within the grouping of similar records. The master
+record acts as the base at which any additional information from similar records are merged into.
+
+Optionally, no master record can be set, leaving Odoo to choose a record to merge into at random.
+
+Next, click the :guilabel:`Merge` button at the top of the similar records grouping. Then click on
+:guilabel:`Ok` to confirm the merge.
+
+.. tip::
+   Discard groupings by clicking the :guilabel:`DISCARD` button. Upon doing so, the grouping is
+   hidden from the list and archived.
+
+   View discarded groupings by selecting the :guilabel:`Discarded` filter from the :ref:`Search...
+   <search/filters>` bar.
+
+Once a record is merged, a message is logged in the chatter of the master record describing the
+merge. Certain records, like *Project* tasks, are logged in the chatter with a link to the old
+record for a convenient reference.
+
+.. _data_cleaning/deduplication-rules:
+
+Deduplication rules
+-------------------
+
+The *Deduplication rules* (:menuselection:`Data Cleaning --> Configuration --> Rules:
+Deduplication`) set the conditions for records to be detected as duplicates.
+
+These rules can be configured for each model in the database, and with varying levels of
+specificity.
+
+The deduplication rules run every hour by default as part of a scheduled action chron (*Data Merge:
+Find Duplicate Records*), however, each rule can be  :ref:`ran manually
+<data-cleaning/run-deduplication-rule>` anytime.
+
+Modify a deduplication rule
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Select a default rule to edit, or create a new rule by clicking on the :guilabel:`New` button.
+
```

Comment:
Yes– except the entered title is replaced once a **Model** is selected... so I decided to just explain selecting a Model as the first step, since this will add a default title that the user can _then_ replace/update :)